<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaSacar</class>
 <widget class="QMainWindow" name="TelaSacar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>641</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="user_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(192, 97, 203);</string>
      </property>
      <property name="text">
       <string>SACAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="inputvalor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(192, 97, 203);</string>
      </property>
      <property name="text">
       <string>Digite o valor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="valor_Sacar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(192, 97, 203);
color: rgb(145, 65, 172);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_sacar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(145, 65, 172);</string>
      </property>
      <property name="text">
       <string>Sacar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_voltar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>480</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Voltar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
